suites/message/create_actor.go

```diff
@@ -116,8 +116,8 @@ func TestInitActorSequentialIDAddressCreate(t *testing.T, factory state.Factorie
 	firstPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+1)
 	secondPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+2)
 
-	firstInitRet := td.ComputeInitActorExecReturn(senderID, 0, firstPaychAddr)
-	secondInitRet := td.ComputeInitActorExecReturn(senderID, 1, secondPaychAddr)
+	firstInitRet := td.ComputeInitActorExecReturn(senderID, 1, firstPaychAddr)
+	secondInitRet := td.ComputeInitActorExecReturn(senderID, 2, secondPaychAddr)
```

Comment:
Sorry I didn't notice this change first review. I don't think this is right, `0` and `1` are the CallSeqNum of the from (`sender`) Actor. Actors are initialized with `CallSeqNum: 0` and `sender` shouldn't have had their `CallSeqNum` incremented at this point, unless I am missing something?

Reply:
If I'm understanding the spec correctly, they *should* have it incremented by this point. I'm basing that off the `ApplyMessage` function here: https://filecoin-project.github.io/specs/#vminterpreter-implementation

AFAIC tell, the rule is that you increment the nonce as soon as the basic validity checks have passed, and *before* any actual actor code is invoked.

Whether that *should* be the rule is ofc up for debate, but rn it seems that the spec says that the `CallSeqNum`s should already have been incremented.

Reply:
That is what the spec code says, indeed. It wasn't consciously that way. [Discussion](https://filecoinproject.slack.com/archives/CHMNDCK9P/p1583875804246100)